invisible_flow/copa/loader.py

```diff
@@ -7,24 +8,23 @@
 class Loader:
 
     def __init__(self):
-        self.existing_crid = pd.DataFrame(
-            DataAllegation.query.with_entities(DataAllegation.cr_id)
-        ).values.flatten().tolist()
         self.matches = []
         self.new_data = []
 
     def load_into_db(self, transformed_data: pd.DataFrame):
 
         for row in transformed_data.itertuples():
 
-            if row.cr_id not in self.existing_crid:
-                new_allegation = DataAllegation(cr_id=row.cr_id)
-                db.session.add(new_allegation)
-                self.new_data.append(transformed_data.iloc[row[0]])
-            else:
+            new_allegation = DataAllegation(cr_id=row.cr_id)
+            db.session.add(new_allegation)
+            try:
+                db.session.commit()
+            except IntegrityError:
                 self.matches.append(transformed_data.iloc[row[0]])
```

Comment:
I see that we are gathering all the data that is already in the db into this variable. Do we later log them somewhere?
Should we consider logging here?